src/graph_building/tests/test_graph_gen.py

```diff
@@ -0,0 +1,99 @@
+import pandas as pd
```

Comment:
A hiányzó adattal nem kell foglalkozni a tesztelésnél - a jelenlegi adatban talán nincs hiányzó érték (?), ha meg lenne, akkor arra előtte rá kell engedni egy "forward fill"-t (`pandas` metódus a `fill`).

Cserébe fontos, hogy a "plató" jelenség kezelve legyen, szóval az, azaz az `[1, 2, 3, 3, 3, 3]` esetén egyetlen csúcsot találunk.

A tesztesetek nézhetnek ki valahogy így:
- egyetlen lokális maximum
- konstans
- platózás
- "platós" csúcs (`1, 2, 3, 3, 2, 1`)
- két lokális maximum, amiből az egyik lesz csak csúcs